lib/tasks_private/add_user.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# This script adds a user to the Azure dev environment using their Redhat
+# userid. By default, it will create a resource group in the east US region
+# with the same name as the userid, scope the user to that resource group,
+# and the credentials will be good for two years.
+
+# The password will only be displayed once, so be sure to save it somewhere.
+
+# Usage:
+#
+#   add_user.sh some_user [eastus 12345]
+#
+
+username=$1
+region="${2:-eastus}"
+subscription_id="${3:-$AZURE_SUBSCRIPTION_ID}"
+
+if [ -z "$username" ]; then
+  echo "You must specify a username argument. This should match the user's Redhat userid."
```

Comment:
> This should match the user's Redhat userid.

Don't think we should have this ^ upstream

Reply:
Changed "Redhat" to "company".